package.json

```diff
@@ -9,11 +9,14 @@
   "scripts": {
     "start": "./bin/server",
     "cover": "nyc mocha test/",
-    "posttest": "npm run eslint",
+    "prepublishOnly": "npm run docs",
+    "pretest": "npm run docs",
     "test": "npm run cover && npm run report",
+    "posttest": "npm run eslint",
     "report": "nyc report --reporter=lcov",
     "eslint": "eslint-godaddy -c .eslintrc lib/ test/",
-    "sandbox": "npm-registry-echo"
+    "sandbox": "npm-registry-echo",
+    "docs": "swagger-jsdoc -d lib/swagger.js -o lib/swagger.json"
```

Comment:
Nitpick: can we name this `swagger`? or `openapi`? The name `docs` makes me think of actual deep documentation (like compiling a static site).

Reply:
Will make it `swagger` since that's what the library is called.